mw-webapp/src/logic/aboutProjectPage/TeamMember/TeamMember.tsx

```diff
@@ -0,0 +1,99 @@
+import {Avatar, AvatarSize} from "src/component/avatar/Avatar";
+import styles from "src/logic/aboutProjectPage/TeamMember/TeamMember.module.scss";
+
+/**
+ *TeamMemberType
+ */
+export interface TeamMemberType {
+
+  /**
+   * Id member
+   */
+  id: number;
+
+  /**
+   * Name member
+   */
+  name: string;
+
+  /**
+   * Profession member
+   */
+  profession: string;
+
+  /**
+   * Image Url
+   */
+  imageUrl: string;
+}
+
+export const teamMembers: TeamMemberType[] = [
```

Comment:
it is a static variable, please move to separate file

Reply:
done